MusicTaste/MusicTaste/User.m

```diff
@@ -0,0 +1,44 @@
+#import "User.h"
```

Comment:
Great example of a model class. Have you considered using model classes for Spotify genres, artists, etc?

Reply:
I have considered that but I feel like it will be more convenient to retrieve all the information through Parse because its only called in Matching Algorithm